rust/extractor/src/translate/base.rs

```diff
@@ -687,6 +689,14 @@ impl<'a> Translator<'a> {
             {
                 return true;
             }
+            if syntax
```

Comment:
I think we can simply drop any node of type TokenTree. That includes macro call arguments, as well as macro rule/definition bodies. Like done in https://github.com/github/codeql/pull/19581